packages/liveblocks-core/turbo.json

```diff
@@ -0,0 +1,19 @@
+{
+  // Extend the global Turbo file
+  "extends": ["//"],
+
+  "pipeline": {
+    "test:e2e": {
+      "dependsOn": ["build"],
+      "inputs": [
+        "jest.*",
+        "e2e/**/*.tsx",
+        "e2e/**/*.ts",
+        "src/**/*.tsx",
+        "src/**/*.ts",
+        "test/**/*.ts",
+        "test/**/*.tsx"
+      ]
+    }
```

Comment:
I've also added this missing definition to the `turbo.json` file. I've used a local `turbo.json` file here, since this command only makes sense inside the `liveblocks-core` package.